lib/ansible/module_utils/basic.py

```diff
@@ -1879,10 +1879,17 @@ def _load_params(self):
 
     def _log_to_syslog(self, msg):
         if HAS_SYSLOG:
-            module = 'ansible-%s' % self._name
-            facility = getattr(syslog, self._syslog_facility, syslog.LOG_USER)
-            syslog.openlog(str(module), 0, facility)
-            syslog.syslog(syslog.LOG_INFO, msg)
+            try:
+                module = 'ansible-%s' % self._name
+                facility = getattr(syslog, self._syslog_facility, syslog.LOG_USER)
+                syslog.openlog(str(module), 0, facility)
+                syslog.syslog(syslog.LOG_INFO, msg)
+            except TypeError as e:
+                self.fail_json(
+                    msg='Failed to log to syslog: %s' % to_native(e),
+                    exception=traceback.format_exc(),
+                    msg_to_log=msg,
```

Comment:
point at DEFAULT_NO_TARGET_SYSLOG to disable remote logging to syslog

Reply:
Dammit, I had that change and forgot to commit it. Done.